e2e/cypress/integration/messaging/MM-T134-visual-verification-of-tooltips_spec.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+const timestamp = Date.now();
+
+function verifyChannel(res, verifyExistence = true) {
+    const channel = res.body;
+
+    // # Hover on the channel name
+    cy.get(`#sidebarItem_${channel.name}`).should('be.visible').trigger('mouseover');
+
+    // * Verify that the tooltip is displayed
+    if (verifyExistence) {
+        cy.get('div.tooltip-inner').
+            should('be.visible').
+            and('contain', channel.display_name);
+    } else {
+        cy.get('div.tooltip-inner').should('not.exist');
+    }
+}
+
+describe('Messaging', () => {
+    let loggedUser;
+
+    let testTeam;
+
+    before(() => {
+        // # Login as test user and visit the newly created test channel
+
+        cy.apiInitSetup({loginAfter: false}).then(({team, user}) => {
+            testTeam = team;
+            loggedUser = user;
+
+            cy.apiAdminLogin;
+
+            cy.apiAddUserToTeam(testTeam.id, loggedUser.id);
+
+            cy.visit(`/${testTeam.name}/channels/town-square`);
+        });
+    });
+    it('MM-T134-visual-verification-of-tooltips', () => {
+        // * Hover effect wraps around Date and New Messages lines
+        // supposed to validate shadow over effect wraping around date but not sure if this is possible
+        cy.get('#postListContent').find('.top').should('be.visible');
+
+        // * Members tool-tip is present
+        cy.findAllByLabelText('members').should('be.visible').trigger('mouseover');
+        cy.get('div.tooltip-inner').should('be.visible').and('contain', 'Members');
+        cy.findAllByLabelText('members').should('be', 'visible').trigger('mouseout');
```

Comment:
Getting an error. The correct way to assert is:

`        cy.findAllByLabelText('members').should('be.visible').trigger('mouseout');`

Please change this throughout the test